gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/services/CgmesExportService.java

```diff
@@ -176,11 +177,22 @@ private Map<String, ByteArrayOutputStream> createOneSsh(Network network, SweData
             addCgmesIdToMap(tso, reporterSsh);
             String filenameFromCgmesExport = network.getNameOrId() + "_SSH.xml";
             baos.write(memDataSource.getData(filenameFromCgmesExport));
-            String newFileName = buildCgmesFilename(sweData, tso, "SSH");
+            CgmesSshMetadata cgmesSshMetadata = network.getExtension(CgmesSshMetadata.class);
+            String sshVersionInFileName = null;
+            if (cgmesSshMetadata != null) {
+                sshVersionInFileName = getFormattedVersionString(cgmesSshMetadata.getSshVersion());
+            } else {
+                sshVersionInFileName = DEFAULT_VERSION;
```

Comment:
This block could be simpler : 
String sshVersionInFileName = cgmesSshMetadata != null ? getFormattedVersionString(cgmesSshMetadata.getSshVersion()) : DEFAULT_VERSION;